src/Middleware/StaticFiles/test/UnitTests/StaticFileMiddlewareTests.cs

```diff
@@ -201,6 +201,139 @@ private async Task FoundFile_Served(string baseUrl, string baseDir, string reque
         }
     }
 
+    [Fact]
+    public async Task OnPrepareResponse_Executed_Test()
+    {
+        var baseUrl = "";
+        var baseDir = @".";
+        var requestUrl = "/testDocument.Txt";
```

Comment:
Linux case sensitivity issue. The current version was copied from a Windows only test.
```suggestion
        var requestUrl = "/TestDocument.txt";
```